scripts/compile-ajv-validators.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable */
```

Comment:
Don't disable eslint. In this case, you probably want node:

```suggestion
/* eslint-env node */
```